smac/runhistory/runhistory2epm.py

```diff
@@ -249,53 +298,63 @@ def _build_matrix(self, run_list, runhistory, instances=None):
             else:
                 X[row, :] = conf.get_array()
             # run_array[row, -1] = instances[row]
-            y[row, 0] = run.cost
-
-        #ensure that minimal value is larger than 0
-        if np.sum(y<=0) > 0:
-                self.logger.warning("Got cost of smaller/equal to 0. Replace by %f since we use log cost." %(constants.MINIMAL_COST_FOR_LOG))
-                y[y<constants.MINIMAL_COST_FOR_LOG] = constants.MINIMAL_COST_FOR_LOG
-        y = np.log10(y)
+            if self.scenario.run_obj == "runtime":
+                if run.status != StatusType.SUCCESS:
```

Comment:
Unfortunately, we cannot simplify it. 
Sometimes we need this function with `par_factor=10` and sometimes with `par_factor=1`, e.g. for the imputation of censored data. Please don't ask me why this is the case.